packages/SwingSet/misc-tools/vat-map-from-swingstore.js

```diff
@@ -0,0 +1,155 @@
+// @ts-nocheck
+import '@endo/init';
+import sqlite3 from 'better-sqlite3';
+import fs from 'fs';
+import process from 'process';
+
+// run this like:
+//  node vat-map-from-swingstore.js run-04-swingstore-pruned.sqlite vat-map-04.json 04
```

Comment:
What is the significance of "pruned" here?

Reply:
I've got a script (for my follower node) which deletes non-current transcript items from the DB, and stores the result in a file with `pruned.sqlite` in the name. I use those pruned ones for analysis work. Once the upgrade17 mostly-automatic-pruning code is deployed (and I change my follower to use it), I can stop using this script, and all my follower's saved swingstores will have the same name.

But yeah, it doesn't have any effect on this tool, I'll remove that string to avoid confusion.